<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjedCheck</class>
 <widget class="QMainWindow" name="ObjedCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ObjedCheck</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/objedcheck.qrc">
    <normaloff>:/mainIcon/objedcheck</normaloff>:/mainIcon/objedcheck</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="centralLayout">
    <item>
     <widget class="QGraphicsView" name="preview"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openImageDirAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="openImageViewAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="actionsMenu">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="nextImageAction"/>
    <addaction name="previousImageAction"/>
    <addaction name="firstImageAction"/>
    <addaction name="lastImageAction"/>
    <addaction name="separator"/>
    <addaction name="refreshAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="actionsMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="imageListDock">
   <property name="windowTitle">
    <string>Image List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="imageListContents">
    <layout class="QHBoxLayout" name="imageListLayout">
     <item>
      <widget class="QListWidget" name="imageList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="propertiesDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="classifierContents">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="detectorLabel">
       <property name="text">
        <string>Detector:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="detectorLayout">
       <item>
        <widget class="QLineEdit" name="detectorLine">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="chooseDetectorButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="clearDetectorButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="minPowerLabel">
       <property name="text">
        <string>Minimum Power:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="minPowerBox">
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="consoleDock">
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="consoleContents">
    <layout class="QHBoxLayout" name="consoleLayout">
     <item>
      <widget class="QTextEdit" name="console">
       <property name="font">
        <font>
         <family>Courier New</family>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="../res/objedcheck.qrc">
     <normaloff>:/actionIcons/about</normaloff>:/actionIcons/about</iconset>
   </property>
   <property name="text">
    <string>&amp;About ObjedCheck</string>
   </property>
  </action>
  <action name="nextImageAction">
   <property name="icon">
    <iconset resource="../res/objedcheck.qrc">
     <normaloff>:/actionIcons/nextImage</normaloff>:/actionIcons/nextImage</iconset>
   </property>
   <property name="text">
    <string>&amp;Next Image</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="previousImageAction">
   <property name="icon">
    <iconset resource="../res/objedcheck.qrc">
     <normaloff>:/actionIcons/previousImage</normaloff>:/actionIcons/previousImage</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous Image</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="firstImageAction">
   <property name="icon">
    <iconset resource="../res/objedcheck.qrc">
     <normaloff>:/actionIcons/firstImage</normaloff>:/actionIcons/firstImage</iconset>
   </property>
   <property name="text">
    <string>&amp;First Image</string>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="lastImageAction">
   <property name="icon">
    <iconset resource="../res/objedcheck.qrc">
     <normaloff>:/actionIcons/lastImage</normaloff>:/actionIcons/lastImage</iconset>
   </property>
   <property name="text">
    <string>&amp;Last Image</string>
   </property>
   <property name="shortcut">
    <string>Shift+D</string>
   </property>
  </action>
  <action name="refreshAction">
   <property name="icon">
    <iconset resource="../res/objedcheck.qrc">
     <normaloff>:/actionIcons/redrawImage</normaloff>:/actionIcons/redrawImage</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="openImageDirAction">
   <property name="icon">
    <iconset resource="../res/objedcheck.qrc">
     <normaloff>:/actionIcons/openImage</normaloff>:/actionIcons/openImage</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Image Directory...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="openImageViewAction">
   <property name="text">
    <string>Open Image View...</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>imageList</tabstop>
  <tabstop>detectorLine</tabstop>
  <tabstop>chooseDetectorButton</tabstop>
  <tabstop>clearDetectorButton</tabstop>
  <tabstop>console</tabstop>
  <tabstop>preview</tabstop>
 </tabstops>
 <resources>
  <include location="../res/objedcheck.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutAction</sender>
   <signal>triggered()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onAboutAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openImageDirAction</sender>
   <signal>triggered()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onOpenImageDirAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstImageAction</sender>
   <signal>triggered()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onChangeImageAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastImageAction</sender>
   <signal>triggered()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onChangeImageAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextImageAction</sender>
   <signal>triggered()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onChangeImageAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousImageAction</sender>
   <signal>triggered()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onChangeImageAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshAction</sender>
   <signal>triggered()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onRefreshAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageList</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ObjedCheck</receiver>
   <slot>processImage(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openImageViewAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onOpenImageViewAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseDetectorButton</sender>
   <signal>clicked()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onChooseDetectorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearDetectorButton</sender>
   <signal>clicked()</signal>
   <receiver>ObjedCheck</receiver>
   <slot>onClearDetectorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAboutAction()</slot>
  <slot>onOpenImageDirAction()</slot>
  <slot>onChangeImageAction()</slot>
  <slot>onRefreshAction()</slot>
  <slot>processImage(QString)</slot>
  <slot>onOpenImageViewAction()</slot>
  <slot>onChooseDetectorClicked()</slot>
  <slot>onClearDetectorClicked()</slot>
 </slots>
</ui>
